code/game/turfs/simulated/floor/floor_types_eris.dm

```diff
@@ -883,6 +883,8 @@
 		/obj/structure/flora/ausbushes/sparsegrass,
 		/obj/structure/flora/ausbushes/fullgrass
 		)
+
+CREATE_STANDARD_TURFS(/turf/simulated/floor/outdoors/grass/heavy)
```

Comment:
why not repath it to /floor/grass/heavy?
(/floor/outdoors/grass/heavy/outdoors looks stupid)

Reply:
/turf/simulated/floor/outdoors has some extra functionality tied to it that for the moment Im not looking to address. At some it should probably just be renamed to be a bit less stupid and to not have outdoors tied to it